tests/test_jobs/test_import_asc_athletes.py

```diff
@@ -51,8 +53,8 @@ def test_update_safety_check(self, app):
         }
         modified_response = MockResponse(200, {}, json.dumps(skinny_import))
         with register_mock(asc_athletes_api._get_asc_feed_response, modified_response):
-            status = ImportAscAthletes().run()
-            assert status is False
+            with pytest.raises(BackgroundJobError):
+                ImportAscAthletes().run()
```

Comment:
@paulkerschen should there be an assertion here? Same below.

Reply:
@alyttle In this case `pytest.raises` is acting as an assertion; the test will fail if it doesn’t hit an exception of that class.

https://docs.pytest.org/en/latest/assert.html#assertraises

Reply:
Makes sense!